.travis.yml

```diff
@@ -2,11 +2,25 @@ language: python
 python: 3.5
 env:
   - TOXENV=py27
+  - TOXENV=pypy
   - TOXENV=py33
   - TOXENV=py34
   - TOXENV=py35
 
 install:
+  - |
+      if [ "$TOXENV" = "pypy" ]; then
+        export PYENV_ROOT="$HOME/.pyenv"
+        if [ -f "$PYENV_ROOT/bin/pyenv" ]; then
+          pushd "$PYENV_ROOT" && git pull && popd
+        else
+          rm -rf "$PYENV_ROOT" && git clone --depth 1 https://github.com/yyuu/pyenv.git "$PYENV_ROOT"
+        fi
+        export PYPY_VERSION="5.3.1"
```

Comment:
I could not find it in pyenv docs (I haven't searched much to be honest), but there must be a way